docs/book_files/csi_block_storage_kc_pdfs.md

```diff
@@ -19,7 +19,7 @@ IBM resources
 
 -   [IBM SAN Volume Controller documentation](https://www.ibm.com/docs/en/sanvolumecontroller)
 -   [IBM Spectrum Scale documentation](https://www.ibm.com/docs/en/spectrum-scale)
--   [IBM FlashSystem® 5200, 5000, 5100, Storwize® V5100 and V5000E documentation](http://www.ibm.com/docs/en/f555sv-and-v)
+-   [IBM FlashSystem® 5200, 5000, 5100, Storwize® V5100 and V5000E documentation](http://www.ibm.com/docs/en/f5x00)
```

Comment:
this link doesn't work for me either

Reply:
i'm looking into this with the SVC team.

Reply:
any update?

Reply:
they're having a technical issue on their end. i'm doublechecking that i've updated correctly though so that we go out properly and when they fix it the link will work